### DISCUSSION TITLE
List of hot keys

### DISCUSSION THREAD

**Discussion by b-user** [tnodir/fort#172] [Q&A]
In the program settings there is a mention of hot buttons.
Is there a document (wiki page) with a list of these hot buttons?
Including hot buttons for clicking on the tray icon.

**Comment by tnodir** [comment]
Hot keys defined here:
https://github.com/tnodir/fort/blob/471992a5c6db1b12af0baf1465c9a76ff43a2350/src/ui/user/iniuser.h#L26

You can edit the `FortFirewall.user.ini` by hand to set custom hot-keys. Example:
```
[hotKey]
home=Ctrl+Alt+Shift+H
programs=Ctrl+Alt+Shift+P
```
